build/azDevOps/azure/ensono/release-notes/publish-release-notes.yml

```diff
@@ -1,55 +1,55 @@
+  - ${{ if eq(parameters.enable, true) }}:
+      - stage: ReleaseNotes
+        dependsOn:
+          - Dev
+          - Prod
+        condition: or(succeeded('Dev'), succeeded('Prod'))
+        variables:
+          - name: core_environment
+            value: $[ coalesce(stageDependencies.Dev.DeployDev.outputs['DeployDev.SetEnv.core_environment'], stageDependencies.Prod.DeployProd.outputs['DeployProd.SetEnv.core_environment']) ]
+        jobs:
+          - job: GenerateAndPublish
+            uses:
+              repositories: [wiki] # work around for security issue for pipelines to access the WIKI repo
+            steps:
+              - task: XplatGenerateReleaseNotes@4
+                displayName: Generate Release Notes
+                inputs:
+                  outputfile: "$(Build.ArtifactStagingDirectory)/releasenotes.md"
+                  templateLocation: "File"
+                  templatefile: "$(System.DefaultWorkingDirectory)/build/azDevOps/azure/ensono/release-notes/handlebar/releaseNoteTemplate.handlebars"
+                  customHandlebarsExtensionFile: "custom-extensions.js"
+                  customHandlebarsExtensionFolder: "$(System.DefaultWorkingDirectory)/build/azDevOps/azure/ensono/release-notes/handlebar/extensions"
+                  checkStage: false
+                  replaceFile: false
+                  getParentsAndChildren: true
+                  getAllParents: false
+                  getIndirectPullRequests: false
+                  stopOnError: true
+                  considerPartiallySuccessfulReleases: true
+                  checkForManuallyLinkedWI: true # Picks up items manually linked rather than via commit message
+                  wiqlFromTarget: "WorkItems"
+                  dumpPayloadToFile: true
+                  dumpPayloadFileName: $(Build.ArtifactStagingDirectory)/release-notes-payload.json
 
-      - task: WikiUpdaterTask@3
-        displayName: Publish Release Notes to WIKI
-        inputs:
-          repo: "https://ensonodigitaluk@dev.azure.com/ensonodigitaluk/Stacks/_git/Stacks.wiki"
-          branch: "wikiMaster"
-          filename: "Releases/Release-Notes/Release-$(version_major).$(version_minor).md"
-          replaceFile: false # appends to the end of the existing WIKI
-          dataIsFile: true
-          sourceFile: "$(Build.ArtifactStagingDirectory)/releasenotes.md" # file output from previous task
-          message: "Automatic commit from pipeline"
-          gitname: "pipeline"
-          gitemail: "pipeline@example.com" # this should be a variable
-          useAgentToken: true
-          localpath: '$(System.DefaultWorkingDirectory)\repo'
-          updateOrderFile: true
-          prependEntryToOrderFile: true
-          injecttoc: true
+              - task: WikiUpdaterTask@3
+                displayName: Publish Release Notes to WIKI
+                inputs:
+                  repo: "https://ensonodigitaluk@dev.azure.com/ensonodigitaluk/Stacks/_git/Stacks.wiki"
```

Comment:
Is there any way we can not hard-code this? Not sure if there is a way of referencing the repo resource or something.

Reply:
Good idea, switched to referencing the wiki repo resource